english/candyland/rainbow/rainbow.md

```diff
@@ -15,6 +15,6 @@ Yellow taste like barf, and green taste like boogers.
 
 You remember your past and feel ashamed you know what those taste like. 
 
-You begin to cry uncontrollably.
+You begin to cry uncontrollably.  But, everyone was five once, so you get over it, dry your tears, and vow never to lick another rainbow.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.